diff -ruN unpatched/zfec-1.2.0-3/debian/changelog tmpdir/zfec-1.2.0-3/debian/changelog
--- unpatched/zfec-1.2.0-3/debian/changelog	1969-12-31 17:00:00.000000000 -0700
+++ tmpdir/zfec-1.2.0-3/debian/changelog	2007-11-27 09:39:18.000000000 -0700
@@ -0,0 +1,5 @@
+zfec (1.2.0-3) unstable; urgency=low
+
+  * debs
+
+ -- alen <alen@ubuntu-box>  Thur, 22 Nov 2007 20:12:47 -0600
diff -ruN unpatched/zfec-1.2.0-3/debian/compat tmpdir/zfec-1.2.0-3/debian/compat
--- unpatched/zfec-1.2.0-3/debian/compat	1969-12-31 17:00:00.000000000 -0700
+++ tmpdir/zfec-1.2.0-3/debian/compat	2007-11-27 09:39:18.000000000 -0700
@@ -0,0 +1 @@
+4
diff -ruN unpatched/zfec-1.2.0-3/debian/control tmpdir/zfec-1.2.0-3/debian/control
--- unpatched/zfec-1.2.0-3/debian/control	1969-12-31 17:00:00.000000000 -0700
+++ tmpdir/zfec-1.2.0-3/debian/control	2007-11-27 09:39:18.000000000 -0700
@@ -0,0 +1,12 @@
+Source: zfec
+Section: python
+Priority: optional
+Maintainer: Alen Peacock <apeacock@flud.org>
+Build-Depends: debhelper (>= 5.0.0), python-dev, devscripts, autoconf, automake
+Standards-Version: 3.6.1
+
+Package: zfec
+Architecture: i386
+Description: zfec 
+Depends: python, pyutil (>= 1.3.6), argparse (>= 0.8.0)
+
diff -ruN unpatched/zfec-1.2.0-3/debian/rules tmpdir/zfec-1.2.0-3/debian/rules
--- unpatched/zfec-1.2.0-3/debian/rules	1969-12-31 17:00:00.000000000 -0700
+++ tmpdir/zfec-1.2.0-3/debian/rules	2007-11-27 09:39:18.000000000 -0700
@@ -0,0 +1,60 @@
+#!/usr/bin/make -f
+
+#export DH_VERBOSE=1
+#export DH_OPTIONS=-v
+
+configure: configure-stamp
+configure-stamp:
+	dh_testdir
+	touch configure-stamp
+
+build: build-stamp
+build-stamp: configure-stamp
+	dh_testdir
+	touch build-stamp
+
+clean:
+	dh_testdir
+	dh_testroot
+	rm -f build-stamp configure-stamp
+
+	-python setup.py clean --all
+
+	dh_clean
+
+install: build
+	dh_testdir
+	dh_testroot
+	dh_clean -k
+	dh_installdirs
+	dh_install
+
+	python setup.py install --root=$(CURDIR)/debian/zfec
+	#python setup.py clean --all
+
+binary-indep: build install
+
+binary-arch: build install
+	dh_testdir
+	dh_testroot
+	dh_installchangelogs
+	dh_installdocs
+	dh_installexamples
+	dh_installlogrotate
+	#dh_installinit -pzfec --name zfec -r --no-start -- defaults 90
+	#dh_installcron
+	#dh_installman
+	dh_link
+	dh_strip
+	dh_compress
+	dh_fixperms
+	#dh_pysupport
+	dh_installdeb
+	dh_shlibdeps
+	dh_gencontrol
+	dh_md5sums
+	dh_builddeb
+
+binary: binary-indep binary-arch
+.PHONY: build clean binary-indep binary-arch binary install configure
+
diff -ruN unpatched/zfec-1.2.0-3/setup.cfg tmpdir/zfec-1.2.0-3/setup.cfg
--- unpatched/zfec-1.2.0-3/setup.cfg	1969-12-31 17:00:00.000000000 -0700
+++ tmpdir/zfec-1.2.0-3/setup.cfg	2007-11-27 09:23:58.000000000 -0700
@@ -0,0 +1,11 @@
+[install]
+optimize = 1
+
+[bdist_rpm]
+# need do '%_unpackaged_files_terminate_build 0' in ~/.rpmmacros for fedora
+release = 1
+packager = Alen Peacock <apeacock@flud.org>
+requires =  python 
+			pyutil >= 1.3.5
+			argparse >= 0.8
+build_requires = python-setuptools
